dev_guide/events.adoc

```diff
@@ -45,3 +45,172 @@ This section will be expanded as more information becomes available
 
 // TODO: out-of-band example
 ////
+
+[[events-reference]]
+== Comprehensive List of Events
+
+// TODO:
+// - Consult various source code; create tables.
+//   - Kubernetes
+//     pkg/kubelet/container/event.go
```

Comment:
This is a good place to start however it is not a comprehensive list since not all events are using these constants.  In order to find all events I would look for usages of `Event`, `Eventf`, and `PastEventf` methods that are defined in pkg/client/record/event.go

cc @ncdc